### PR TITLE
[REFORMAT] Style Upgrade

### PR DESCRIPTION
**Pull request by mrmundt**
## Description
The original site template is good for a workshop, but it is not sufficient for a conference website that has a lot more information. I mimicked the style of the main US-RSE website here instead and reformatted to have logical dropdowns for different categories.

## Motivation and Context
We want the site to look professional.

## Checklist:

- [x] (Committee Chairs Only) I have posted the link for the PR in the usrse slack (#usrse-2023-conference-planning-committee) to ask for content reviewers
- [x] I have previewed changes locally
- [ ] I have updated the README.md if necessary

cc @mrmundt @cabejackson @manning-ncsa @jmelot @jbteves @jscubida

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,259 +1,87 @@
-# --- General options --- #
+# Site Settings
+lang: en
+
+title: US-RSE'23
+description: The first US-RSE Conference ever!
 
-# url is the full website URL
-# baseurl is the website's URL without the hostname
-# DO NOT CHANGE THE LINE NUMBERS HERE without changing .circleci/circle_urls.sh
```
Comment (jsubida) [inline _config.yml:5]: @mrmundt Would the changes in this file affect CircleCI?

**Review by jsubida — CHANGES_REQUESTED** [review]
Just wanted to raise the config file issue. But otherwise looks good to me!

File: _config.yml (hunk 1 above)
Comment (mrmundt) [reply _config.yml:5]: Whoops, that's residual from copy-pasta-ing from the main US-RSE repository. We don't have CircleCI set up on this repo.

**Comment by jdamerow** [comment]
Looks good to me! Only thing, I'm not sure if we should put the CI line in the footer, since CI is not (legally) involved at this point.

**Comment by mrmundt** [comment]
> Looks good to me! Only thing, I'm not sure if we should put the CI line in the footer, since CI is not (legally) involved at this point.

Yes, but what the footer says is "US-RSE is a fiscally sponsored project of [Community Initiatives](http://communityin.org/)" <- Not US-RSE'23. So I think it should be fine.

**Review by jsubida — APPROVED** [review]

**Comment by mrmundt** [comment]
Most recent addition (Sponsor page):

Sponsor Page

**Comment by mrmundt** [comment]
@jsubida - If you want to look over it again, feel free - I added the "Sponsor" page.

**Comment by jsubida** [comment]
Looks good! Thanks for making the change to the template.

**Comment by jdamerow** [comment]
> > Looks good to me! Only thing, I'm not sure if we should put the CI line in the footer, since CI is not (legally) involved at this point.
> 
> Yes, but what the footer says is "US-RSE is a fiscally sponsored project of [Community Initiatives](http://communityin.org/)" <- Not US-RSE'23. So I think it should be fine.

Yes, you're probably right. I'm just overly careful. If nobody else thinks, it's a problem, it's probably fine :)

**Review by manning-ncsa — COMMENTED** [review]
I was able to build and run the site locally via Docker and it looks nice overall. However, I think the massive wall-to-wall countdown timer on the landing page is a bit dramatic and distracting from the content.

Screenshot from 2023-03-24 15-56-26

**Comment by mrmundt** [comment]
> I was able to build and run the site locally via Docker and it looks nice overall. However, I think the massive wall-to-wall countdown timer on the landing page is a bit dramatic and distracting from the content.
> 
> Screenshot from 2023-03-24 15-56-26

You don't like to be assaulted by a countdown clock?! ;) I can make it smaller and center it. Please hold.

**Comment by mrmundt** [comment]
Screen Shot 2023-03-24 at 15 16 20

@manning-ncsa - How about this?

**Comment by manning-ncsa** [comment]
> Screen Shot 2023-03-24 at 15 17 56
> 
> @manning-ncsa - How about this?

This is much better, thanks. Verified with a docker build and run locally.

**Pull request merged**